src/cmd/commit.ts

```diff
@@ -177,31 +177,35 @@ export async function checkIfPipelineRunExists(): Promise<void> {
   d.info(`There is a Tekton PipelineRuns continuing...`)
 }
 
-async function createRootCredentialsSecret(credentials: { adminUsername: string; adminPassword: string }) {
-  const secretData = {
-    username: credentials.adminUsername,
-    password: credentials.adminPassword,
-  }
+async function createCredentialsSecret(secretName: string, username: string, password: string): Promise<void> {
+  const secretData = { username, password }
   const kc = new KubeConfig()
   kc.loadFromDefault()
   const coreV1Api = kc.makeApiClient(CoreV1Api)
-  await createGenericSecret(coreV1Api, 'root-credentials', 'default', secretData)
+  await createGenericSecret(coreV1Api, secretName, 'keycloak', secretData)
 }
 
 export const printWelcomeMessage = async (): Promise<void> => {
   const d = terminal(`cmd:${cmdName}:commit`)
   const values = (await hfValues()) as Record<string, any>
-  const credentials = values.apps.keycloak
-  await createRootCredentialsSecret({
-    adminUsername: credentials.adminUsername,
-    adminPassword: credentials.adminPassword,
-  })
+  const { adminUsername, adminPassword }: { adminUsername: string; adminPassword: string } = values.apps.keycloak
+  await createCredentialsSecret('root-credentials', adminUsername, adminPassword)
+  const { hasExternalIDP } = values.otomi
+  const { domainSuffix } = values.cluster
+  const defaultPlatformAdminEmail = `platform-admin@${domainSuffix}`
+  const platformAdmin = values.users.find((user: any) => user.email === defaultPlatformAdminEmail)
+  if (platformAdmin && !hasExternalIDP) {
+    const { email, initialPassword }: { email: string; initialPassword: string } = platformAdmin
+    await createCredentialsSecret('platform-admin-initial-credentials', email, initialPassword)
+  }
+  const secretName = hasExternalIDP ? 'root-credentials' : 'platform-admin-initial-credentials'
   const message = `
   ########################################################################################################################################
   #
-  #  Visit the console at: https://console.${values.cluster.domainSuffix}
-  #  Perform: kubectl get secret root-credentials -n default -o yaml
-  #  To obtain access credentials in base64 encoded format
+  #  Visit the console at: https://console.${domainSuffix}
+  #  Perform: 
```

Comment:
The Application Platform has been succesfully deployed.
Obtain login credentials by using the below command:
<cmd>
, and navigate to https://console.${domainSuffix} where you can manage the platform.
```

Reply:
Updated!